monitor.py

```diff
@@ -1015,10 +1015,12 @@ def get_jobs_with_date(prowci_url,start_date,end_date):
                             job_log_path = ele["SpyglassLink"]
                             final_job_list.append(job_log_path)
 
-                    if next_link_match != None:
-                        next_page_spylink=next_link[35:]
-                        check=get_next_page_first_build_date(next_page_spylink,end_date)
+                    build_regex = r"/([^/?]+)\?.+"
```

Comment:
Can you add comment above the line as how the input will be and what will be the output of this regex?
Also please retitle the PR properly